types/operator_signer.go

```diff
@@ -0,0 +1,76 @@
+package types
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"github.com/pkg/errors"
+)
+
+type OperatorSigner struct {
+	SSVOperatorSK *rsa.PrivateKey
+	OperatorID    OperatorID
```

Comment:
It seems that `OperatorSigner.OperatorID` is unused.

Reply:
removing

Reply:
wait... I see `OperatorSigner.GetOperatorID()` is called in many places...

Reply:
Oh, that's right. I didn't notice the method. Thanks